s4e-web/src/app/views/settings/groups/group-form/group-form.component.ts

```diff
@@ -47,33 +47,40 @@ export class GroupFormComponent extends GenericFormComponent<GroupQuery, GroupFo
     });
 
     (this.form.get('membersEmails').valueChanges as Observable<any>)
-      .pipe(untilDestroyed(this),
+      .pipe(
+        untilDestroyed(this),
         filter(() => this.groupSlug != null),
         distinctUntilChanged()
-      ).subscribe(a => console.log(a));
+      ).subscribe(() => {});
```

Comment:
In this case just calling `.subscribe()` will work, no need to pass empty closure